docs/features/UPSCALE.md

```diff
@@ -97,3 +97,39 @@ the base images.
 If you wish to stop during the image generation but want to upscale or face restore a particular
 generated image, pass it again with the same prompt and generated seed along with the `-U` and `-G`
 prompt arguments to perform those actions.
+
+## CodeFormer Support
+
+This repo also allows you to perform face restoration using
+[CodeFormer](https://github.com/sczhou/CodeFormer).
+
+In order to setup CodeFormer to work, you need to download the models like with GFPGAN. You can do
+this either by running `preload_models.py` or by manually downloading the
+[model file](https://github.com/sczhou/CodeFormer/releases/download/v0.1.0/codeformer.pth) and
+saving it to `ldm/restoration/codeformer/weights` folder.
+
+You can use `-ft` prompt argument to swap between CodeFormer and the default GFPGAN. The above
+mentioned `-G` prompt argument will allow you to control the strength of the restoration effect.
+
+**Usage:**
```

Comment:
Instead of bolding headings, keep piling on the '#' (you just have to keep track of what level you're, which is admittedly a pain :D) - this has special meaning in Markdown, as it creates anchors when e.g. rendering to HTML